app/schemas/org.fossasia.openevent.general.OpenEventDatabase/1.json

```diff
@@ -591,7 +591,7 @@
             "fieldPath": "minOrder",
             "columnName": "minOrder",
             "affinity": "INTEGER",
-            "notNull": false
+            "notNull": true
```

Comment:
These changes should not be in your PR

Reply:
@iamareebjamal https://github.com/fossasia/open-event-android/pull/196
These changes are because of this PR, I think @simarsingh24 forgot to run the app when he changed the nullability here. This is the reason why this appears in every PR sent after this got merged

Reply:
Yeah even in my PR the changes are reflected :+1: